### PR TITLE
Fix for ipyleaflet

### PR DESCRIPTION
**Pull request by TimStewartJ**
Changes css rule for max-width for canvas objects to `unset`, fixes issue where ipyleaflet overlays wouldn't show.

Testing and comparison between the sandboxes on [query](https://query.libretexts.org/?title=Sandboxes%2Fjupyterteam%2540ucdavis.edu) (where ckeditor-binder-plugin uses ipyleaflet-fix branch) and [chem](https://chem.libretexts.org/Sandboxes/jupyterteamucdavisedu) shows no difference between the function of various plugins (with the exception of ipyleaflet of course).

Relevant to issue #107

### CODE REVIEW

**Comment by moorepants** [comment]
Ipycytoscape is not displaying (the canvas element) on query. Is that not effected perhaps?

File: src/styles/plugin.scss [hunk 1]
```diff
@@ -43,6 +43,10 @@
   }
 }
 
+.thebelab-cell canvas {
+  max-width: unset;
+}
+
```
Comment (moorepants) [inline src/styles/plugin.scss:50]: Please add a comment here explaining that this is specifically for allowing ipyleaflet's canvas to work.

You could also check to see if the ipyleaflet canvas uses a specific class name or id name. If so you could make this unset only for ipyleaflet, ensuring it doesn't affect other packages.

**Comment by TimStewartJ** [comment]
I don't think the ipycytoscape issue is related, it also doesnt work on chem libretexts. I also tried using the web dev tools to manually change the canvas css properties for ipycytoscape and couldn't get it to work on either query or chem.

**Comment by sandertyu** [comment]
We thought we fixed ipycytoscape for jupyterlab 2 by pinning it to 1.0.4. This worked with our testing in Thebe originally, but now it doesn't work there either.

Regardless, this fix looks good. The example using the dynamic heatmap from my original issue work on query as well. Looks go to merge.

**Review by sandertyu — APPROVED** [review]

File: src/styles/plugin.scss (hunk 1 above)
Comment (TimStewartJ) [reply src/styles/plugin.scss:50]: Done.

**Pull request merged**